Makefile

```diff
@@ -17,7 +17,7 @@ override LDFLAGS += \
 #  docker image publishing options
 DOCKER_PUSH?=true
 IMAGE_NAMESPACE?=argoproj
-IMAGE_TAG?=v0.8
+IMAGE_TAG?=v0.8.3
```

Comment:
I'd prefer `latest` as the default value so we don't have to update this value every time when we release new versions. What do you think?